File: app/src/main/java/com/example/alkewalletandroid/MainActivity3.java
package com.example.alkewalletandroid;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity3 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main3);

        Button sesion = findViewById(R.id.sesion);
        TextView ncuenta = findViewById(R.id.nueva_cuenta);

        sesion.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity3.this, MainActivity5.class);
                startActivity(intent);
            }
        });

        ncuenta.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent= new Intent(MainActivity3.this, MainActivity4.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/alkewalletandroid/MainActivity3.kt
package com.example.alkewalletandroid

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity3 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main3)
        val sesion = findViewById<Button>(R.id.sesion)
        val ncuenta = findViewById<TextView>(R.id.nueva_cuenta)
        sesion.setOnClickListener {
            val intent = Intent(this@MainActivity3, MainActivity5::class.java)
            startActivity(intent)
        }
        ncuenta.setOnClickListener {
            val intent = Intent(this@MainActivity3, MainActivity4::class.java)
            startActivity(intent)
        }
    }
}